src/vs/editor/browser/viewParts/minimap/minimap.css

```diff
@@ -15,7 +15,18 @@
 	opacity: 1;
 }
 /* END cover the case that slider is visible on mouseover */
-
+.monaco-editor .minimap-slider .minimap-slider-horizontal {
+	color: var(--vscode-minimapSlider-background) !important;
```

Comment:
Do these colours need to have `!important`, considering the original code doesn't use them?

Reply:
@rzhao271 Thank you for the review. 
I edit this.